orders/models.py

```diff
@@ -0,0 +1,57 @@
+from django.db import models
+
+
+class Order(models.Model):
+    user = models.ForeignKey("users.User", on_delete=models.CASCADE)
+    address = models.CharField(max_length=64)
+    memo = models.TextField()
+    order_status = models.ForeignKey("OrderStatus", on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = "orders"
+
+
+class OrderStatus(models.Model):
+    DEPOSIT = "D"
+    ON_DELIVERY = "OD"
+    DELIVERY_COMPLETE = "DC"
+    PURCHASE_COMPLETE = "PC"
+
+    ORDER_CHOICE = [
+        (DEPOSIT, "deposit"),
+        (ON_DELIVERY, "on_delivery"),
+        (DELIVERY_COMPLETE, "delivery_complete"),
+        (PURCHASE_COMPLETE, "purchase_complete")
+    ]
+    status = models.CharField(max_length=32, choices=ORDER_CHOICE)
+
+    class Meta:
+        db_table = "order_status"
+
+
+class OrderProduct(models.Model):
+    product = models.ForeignKey("products.Product", on_delete=models.CASCADE)
+    quantity = models.SmallIntegerField()
+    order = models.ForeignKey("Order", on_delete=models.CASCADE)
+    order_product_status = models.ForeignKey("OrderProductStatus", on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = "order_products"
```

Comment:
주문 상품은 `OrderItem`이라는 용어를 많이 사용합니다~

```suggestion
class OrderItem(models.Model):
    product = models.ForeignKey("products.Product", on_delete=models.CASCADE)
    quantity = models.SmallIntegerField()
    order = models.ForeignKey("Order", on_delete=models.CASCADE)
    status = models.ForeignKey("OrderItemStatus", on_delete=models.CASCADE)

    class Meta:
        db_table = "order_products"
```

Reply:
수정했습니다.